infra/templates/az-isolated-service-single-region/docs/design.md

```diff
@@ -141,7 +141,7 @@ Virtual Service Endpoints (VSE) will be used to restrict access to key services
 |---|---|---|---|
 | Key Vault | [Yes](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-overview-vnet-service-endpoints) | [Yes](https://www.terraform.io/docs/providers/azurerm/r/key_vault.html#network_acls) | [No](https://github.com/microsoft/cobalt/blob/master/infra/modules/providers/azure/keyvault/main.tf) - Missing Configuration |
 | Azure Storage | [Yes](https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?toc=%2fazure%2fvirtual-network%2ftoc.json#grant-access-from-a-virtual-network) | [Yes](https://www.terraform.io/docs/providers/azurerm/r/storage_account.html#virtual_network_subnet_ids) | [No](https://github.com/microsoft/cobalt/pull/183/files) - Missing configuration |
```

Comment:
Are you saying that the Azure Storage module will have to be updated to account for ASE scenarios? If so, how soon?

Reply:
Only in that we need to restrict access to a specific VNET. It will be a small change